test-browser.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+var test = require('tape')
+var leveljs = require('level-js')
+var testCommon = require('level-js/test/util/test-common')
+var levelup = require('levelup')
+var encoding = require('encoding-down')
+
+require('level-packager/abstract/test')(test, require('./'), {
+  skipRepairTest: true,
+  skipErrorIfExistsTest: true,
+  skipDestroyTest: true
+})
```

Comment:
This code came from `test.js`, but using `level-packager/abstract/test` instead.